recipes/fltk/all/conanfile.py

```diff
@@ -0,0 +1,118 @@
+from conans import CMake, ConanFile, tools
+import os
+
+required_conan_version = ">=1.43.0"
+
+class FltkConan(ConanFile):
+    name = "fltk"
+    description = "Fast Light Toolkit is a cross-platform C++ GUI toolkit"
+    topics = ("fltk", "gui")
+    homepage = "https://www.fltk.org"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "LGPL-2.0-custom"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_gl": [True, False],
+        "with_threads": [True, False],
+        "with_gdiplus": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_gl": True,
+        "with_threads": True,
+        "with_gdiplus": True,
+    }
+    # generators = "cmake", "cmake_find_package_multi"
```

Comment:
any reason not to add `cmake_find_package_multi`? it uses CMake, and there are many deps

Reply:
no reason!
I'm afraid I can't recall my disicion.
Thank you reminding me.

Reply:
fixed